modules/UI/videolayout/VideoLayout.js

```diff
@@ -374,24 +378,21 @@ var VideoLayout = {
      * @param id the identifier of the video thumbnail
      */
     handleVideoThumbClicked (id) {
+        const pinnedId = this.getPinnedId();
         var smallVideo = VideoLayout.getSmallVideo(id);
+
         if(pinnedId) {
             var oldSmallVideo = VideoLayout.getSmallVideo(pinnedId);
             if (oldSmallVideo && !interfaceConfig.filmStripOnly) {
                 oldSmallVideo.focus(false);
-                // as no pinned event will be sent for local video
-                // and we will unpin old one, lets signal it
-                // otherwise we will just send the new pinned one
-                if (smallVideo.isLocal)
-                    eventEmitter.emit(
-                        UIEvents.PINNED_ENDPOINT, oldSmallVideo, false);
```

Comment:
This seemed like an unnecessary call now because the redux logic will fire this event later and do pinning logic later on. Maybe there's a case I'm missing...